src/routes/PostDetail/Comments/CommentForm/CommentForm.js

```diff
@@ -88,7 +90,7 @@ export default class CommentForm extends Component {
           contentHTML={editorContent}
         />
 
-        { !currentUser
+        {!currentUser
```

Comment:
@tibetsprague noticing that you like the space before. The space is a new thing and I can embrace that but slightly prefer without the space. Also can not care if it's both ways in the code... :)

Reply:
hmm, dont have a strong preference here